src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$headerTopPadding: 36px;
+$colorDark: #1d1d1d;
```

Comment:
Only two variables for the whole project?
Did you add all the main values? (colors, transition time, main sizes etc.)